### PR TITLE
1.14.0 Release Notes

### PR DESCRIPTION
**Pull request by marccampbell**

### CODE REVIEW

File: content/release-notes/1.14.0.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+* Added a `requireMinimimalRBAC` field in the `apiVersion: k8s.io/v1beta1, Kind=Application` manifest. When enabled and running `kots install`, KOTS will default to a namespaced-scoped RBAC role instead of a cluster-scoped role.
+* Added a new `Distribution` template function to return a string indicating the managed Kubernetes offering (i.e. `openshift`, `eks`, etc).
+{{</features>}}
+
+{{<changes>}}
+* Changed the internal API that `kots upload` and `kots download` use to the Go API. The Typescript implementations are deprecated.
+* Added Velero status to the dashboard tile to identify if Velero is not installed and Snapshots are enabled.
+* When running the Admin Console port forward (after `kots install` or `kots admin-console`), if port 8800 is not available, a random port will be used instead of showing an error.
+* Default support bundle collectors now include rook, ceph, and kurl information.
+* kotsadm-api is given ClusterRole access by default.
```
Comment (marccampbell) [inline content/release-notes/1.14.0.md]: This is not a change, this is how previous versions worked

File: content/release-notes/1.14.0.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+* Added support for a new `additionalNamespaces` field in the `apiVersion: k8s.io/v1beta1, Kind=Application` manifest.
+* Added a `requireMinimimalRBAC` field in the `apiVersion: k8s.io/v1beta1, Kind=Application` manifest. When enabled and running `kots install`, KOTS will default to a namespaced-scoped RBAC role instead of a cluster-scoped role.
+* Added a new `Distribution` template function to return a string indicating the managed Kubernetes offering (i.e. `openshift`, `eks`, etc).
+{{</features>}}
+
+{{<changes>}}
+* Changed the internal API that `kots upload` and `kots download` use to the Go API. The Typescript implementations are deprecated.
+* Added Velero status to the dashboard tile to identify if Velero is not installed and Snapshots are enabled.
+* When running the Admin Console port forward (after `kots install` or `kots admin-console`), if port 8800 is not available, a random port will be used instead of showing an error.
+* Default support bundle collectors now include rook, ceph, and kurl information.
```
Comment (marccampbell) [inline content/release-notes/1.14.0.md]: kurl -> kURL

File: content/reference/template-functions/static-context.md [hunk 3]
```diff
@@ -281,3 +281,11 @@ IsKurl returns true if running within a kurl-based installation.
 repl{{ IsKurl }}
 ```
 
+## Distribution
+```go
+func Distribution() string
+```
+Distribution returns the kubernetes distribution detected, such as `kurl`, `openShift` or `eks`.
```
Comment (marccampbell) [inline content/reference/template-functions/static-context.md]: Can we link to troubleshoot or list all options here?

File: content/release-notes/1.14.0.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+* Added support for a new `additionalNamespaces` field in the `apiVersion: k8s.io/v1beta1, Kind=Application` manifest.
+* Added a `requireMinimimalRBAC` field in the `apiVersion: k8s.io/v1beta1, Kind=Application` manifest. When enabled and running `kots install`, KOTS will default to a namespaced-scoped RBAC role instead of a cluster-scoped role.
+* Added a new `Distribution` template function to return a string indicating the managed Kubernetes offering (i.e. `openshift`, `eks`, etc).
+* Added a new [kURL Context](/reference/template-functions/kurl-context) and corresponding template functions to read 
+values from the kURL Installer CRD. These functions are not available on the config page.
+{{</features>}}
+
+{{<changes>}}
+* Changed the internal API that `kots upload` and `kots download` use to the Go API. The Typescript implementations are deprecated.
+* Added Velero status to the dashboard tile to identify if Velero is not installed and Snapshots are enabled.
```
Comment (marccampbell) [inline content/release-notes/1.14.0.md]: This should have a "When snapshots are enabled"

File: content/release-notes/1.14.0.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+{{<features>}}
+* Added support for a new `additionalNamespaces` field in the `apiVersion: k8s.io/v1beta1, Kind=Application` manifest.
+* Added a `requireMinimimalRBAC` field in the `apiVersion: k8s.io/v1beta1, Kind=Application` manifest. When enabled and running `kots install`, KOTS will default to a namespaced-scoped RBAC role instead of a cluster-scoped role.
+* Added a new `Distribution` template function to return a string indicating the managed Kubernetes offering (i.e. `openshift`, `eks`, etc).
+* Added a new [kURL Context](/reference/template-functions/kurl-context) and corresponding template functions to read 
+values from the kURL Installer CRD. These functions are not available on the config page.
+{{</features>}}
+
+{{<changes>}}
+* Changed the internal API that `kots upload` and `kots download` use to the Go API. The Typescript implementations are deprecated.
```
Comment (marccampbell) [inline content/release-notes/1.14.0.md]: @areed did we remove the enableSnapshots spec field and just look for the presence of a Backup kind in this release? If this is true, will you add a change note?

File: content/release-notes/1.14.0.md (hunk 4 above)
Comment (laverya) [reply content/release-notes/1.14.0.md]: `* Added Velero status to the dashboard tile when Snapshots are enabled to identify if Velero is not installed.`?

File: content/release-notes/1.14.0.md (hunk 5 above)
Comment (areed) [reply content/release-notes/1.14.0.md]: Yes

**Pull request merged**